app/assets/v2/css/grants/grant.css

```diff
@@ -214,27 +213,65 @@
 }
 
 #search_form .sort_container .select2-container {
-  width: 300px !important;
+  width: 270px !important;
 }
 
-#grants_nav{
-  border-bottom: 1px solid #333333;
-  text-align: left;
-  width: 100%;
+#search_form .select2-network-container {
+  width: 5rem !important;
+}
+
+#search_form .network-container .select2-container {
+  min-width: 5rem;
+}
+
+#search_form .select2-selection {
+  background: #F2F6F9;
+  border: none !important;
+  color: var(--gc-blue);
+}
+
+#search_form .select2-selection span {
+  color: #3E00FF;
+  font-weight: bold;
 }
 
-#grants_nav a{
+.grants_type_nav {
+  border-bottom: 1px solid #979797;
+}
+
+.grants_type_nav a.selected {
+  font-weight: bold;
+  border-bottom: 4px solid #3E00FF;
+  padding-bottom: 6px;
+}
+
+.grants_keyword_nav a:hover {
+  border-bottom: 1.5px solid black;
+  padding-bottom: 4px;
+}
+
+.grants_keyword_nav a.selected:hover {
+  border-color: #3E00FF;
+}
+
+.grants_nav a {
   color: #333333;
   margin-right: 30px;
+  cursor: pointer;
 }
 
-#grants_nav a:hover{
-  font-weight: 700;
+.grants_nav a:hover {
+  text-shadow: 0 0 .01px black, 0 0 .01px black, 0 0 .01px black;
+  text-decoration: none;
 }
 
-#grants_nav .selected{
+.select2-selection__arrow {
```

Comment:
lets avoid customising already used classed by page since this will break when we decide to put all in a big css or adding new components to the page having unexpected visual results.

Reply:
https://github.com/gitcoinco/web/pull/5637#discussion_r358298968